src/content/getting_started/architecture/_index.en.md

```diff
@@ -19,3 +20,25 @@ both on-premises and on public clouds:
 Submariner has optional components that provide additional functionality:
 
 * [Globalnet Controller](./globalnet/): handles interconnection of clusters with overlapping CIDRs.
+
+The diagram below illustrates the basic architecture of Submariner:
+
+![Submariner Architecture](/images/submariner/architecture.jpg)
+
+### Terminology and Concepts
+
+* Clusterset - a group of two or more clusters with a high degree of mutual trust that share Services among themselves.
+Within a clusterset, all namespaces with a given name are considered to be the same namespace.
+
+* `ServiceExport` (CRD) - used to specify which Services should be exposed across all clusters in the clusterset. If multiple clusters
+export a Service with the same name and from the same namespace, they will be recognized as a single combined Service.
+
+  * ServiceExports must be explicitly created by the user in each cluster and within the namespace that the underlying Service resides in,
+in order to signify that the Service should be visible and discoverable to other clusters in the clusterset. The `ServiceExport` object can
+be created manually or via the `subctl export` command.
+
+  * When a `ServiceExport` is created, this will cause a domain name of `<service>.<ns>.svc.clusterset.local` for the multi-cluster service
+to become accessible.
```

Comment:
This isn’t a domain name; perhaps

```suggestion
  * When a `ServiceExport` is created, this will cause the multi-cluster service to become accessible as `<service>.<ns>.svc.clusterset.local`.
```